### PR TITLE
Webapp Improvements

### PR DESCRIPTION
**Pull request by nils-wisiol**

### CODE REVIEW

**Comment by nils-wisiol** [comment]
Screenshots:

After registration and creation of a new account and new custom domain:

image

Domain-info in log in area domain list:

image

Option to set account password after registration of dynamic domain and user account:

image

**Comment by peterthomassen** [comment]
Shall I review this, or shall I hold back due to https://github.com/desec-io/desec-stack/issues/502#issuecomment-756938820?

**Comment by nils-wisiol** [comment]
Let's move on with this first

File: webapp/package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "vue": "~2.6.12",
     "vue-clipboard2": "~0.3.1",
     "vue-router": "~3.4.9",
+    "vue-timeago": "^5.1.2",
```
Comment (peterthomassen) [inline webapp/package.json:21]: Then `javascript-time-ago` can go away, right?

File: webapp/src/router/index.js [hunk 2]
```diff
@@ -24,7 +24,8 @@ const routes = [
   {
     path: '/custom-setup/:domain',
     name: 'customSetup',
-    component: () => import(/* webpackChunkName: "signup" */ '../views/CustomSetup.vue')
+    component: () => import(/* webpackChunkName: "signup" */ '../views/DomainSetupPage'),
+    props: true,
```
Comment (peterthomassen) [inline webapp/src/router/index.js:28]: cool!

**Review by peterthomassen — APPROVED** [review]
lgtm! Just one small remark.

Also, I did not visually inspect the subname column width, but I assume it will be sensible.

**Pull request merged**